### ISSUE TITLE
Get in Contact with BlazerExtensions, so that they using this repo

### ISSUE THREAD

**Issue by Ruffo324** [arivera12/BlazorBrowserStorage#1]
Hey nice work! 👍 
~~It would be super nice, if u publish this version to nuget.~~
U can also try to get in Contact with BlazerExtensions, so that they using this repo instead of the dead one.

**Comment by Ruffo324** [comment]
Oh, my bad. I missed the nuget badge in the readme. Sorry.
But it would still be worth a try to inform BlazorExtensions.

**Comment by arivera12** [comment]
Hi, @Ruffo324 I haven't try to contact them, not sure if they want to archive their repository and link to my repository. 

I left a comment [here](https://github.com/BlazorExtensions/Storage/issues/50#issuecomment-635480428).

BlazorExtensions seems to be dead and I crash into that error where I commented so I just made my own version from their code, fixed the issues and improve it removing all javascript dependencies. 

I even made mention of their repository [here](https://github.com/arivera12/BlazorBrowserStorage#no-javascript-library-reference-dependency).

If you wanna help me spread this library it would awesome or try to contact them to link back to my repository it would awesome too.

PS. Don't forget to star my repository and take a look of my other libraries, mot of all of them are based on blazor.
